src/api/health.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { createRequestInstance } from './axiosInstance';
+
+/**
+ * Api calls for healthcheck
+ *
+ * @namespace ApiHealth
+ */
+
+const healthApi = {
+  /**
+   * Get health information of full node running in connected server
+   *
+   * @return {Promise}
+   * @memberof ApiHealth
+   * @inner
+   */
+  async getHealth() {
+    return new Promise((resolve, reject) => {
+      // TODO: We should chage this to get `health` instead of `version`
```

Comment:
This depends on https://github.com/HathorNetwork/hathor-core/pull/807

Reply:
This was merged a week ago. Has it been released already?